src/components/views/messages/MPollBody.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { _t } from '../../../languageHandler';
+import { replaceableComponent } from "../../../utils/replaceableComponent";
+import { IBodyProps } from "./IBodyProps";
+import { IPollAnswer, IPollContent, POLL_START_EVENT_TYPE } from '../../../polls/consts';
+import StyledRadioButton from '../elements/StyledRadioButton';
+
+// TODO: [andyb] Use extensible events library when ready
+const TEXT_NODE_TYPE = "org.matrix.msc1767.text";
```

Comment:
Not sure that's what you meant in the TODO above, but you should probably create a new type using `UnstableValue` in `matrix-js-sdk/src/@types/event.ts`

Reply:
I was copying what Travis did in `src/polls/consts.ts`.  He used `UnstableValue` for e.g. `POLL_START_EVENT_TYPE` but not for `TEXT_NODE_TYPE` - not sure why.